robotmodels/dingo_kinova/urdf/dingo_kinova.urdf

```diff
@@ -0,0 +1,499 @@
+<?xml version="1.0" ?>
+<!-- =================================================================================== -->
+<!-- |    This document was autogenerated by xacro from dingo_kinova.xacro             | -->
+<!-- |    EDITING THIS FILE BY HAND IS NOT RECOMMENDED                                 | -->
+<!-- =================================================================================== -->
+<robot name="dingo_kinova">
+  <!-- Create world joint -->
+  <link name="world"/>
+  <joint name="world_link" type="fixed">
+    <origin rpy="0 0 0" xyz="0 0 0"/>
+    <parent link="world"/>
+    <child link="base_link"/>
+  </joint>
+  <!-- ============================= -->
+  <!-- Omni base links               -->
+  <!-- ============================= -->
+  <link name="base_link">
+    <inertial>
+      <mass value="1.0"/>
+      <origin rpy="0 0 0" xyz="0 0 0"/>
+      <inertia ixx="1.0" ixy="0.0" ixz="0.0" iyy="1.0" iyz="0.0" izz="1.0"/>
+    </inertial>
+  </link>
+  <link name="base_link_x">
+    <inertial>
+      <mass value="1.0"/>
+      <origin rpy="0 0 0" xyz="0 0 0"/>
+      <inertia ixx="1.0" ixy="0.0" ixz="0.0" iyy="1.0" iyz="0.0" izz="1.0"/>
+    </inertial>
+  </link>
+  <link name="base_link_y">
+    <inertial>
+      <mass value="1.0"/>
+      <origin rpy="0 0 0" xyz="0 0 0"/>
+      <inertia ixx="1.0" ixy="0.0" ixz="0.0" iyy="1.0" iyz="0.0" izz="1.0"/>
+    </inertial>
+  </link>
+  <!-- Omni base joints -->
+  <joint name="omni_joint_x" type="prismatic">
+    <safety_controller k_position="100.0" k_velocity="40.0" soft_lower_limit="-500" soft_upper_limit="500"/>
+    <parent link="base_link"/>
+    <child link="base_link_x"/>
+    <origin rpy="0 0 0" xyz="0 0 0.03"/>
+    <axis xyz="1 0 0"/>
+    <dynamics damping="100.0"/>
+    <limit effort="1000" lower="-10" stiffness="1000" upper="10" velocity="2"/>
+  </joint>
+  <joint name="omni_joint_y" type="prismatic">
+    <safety_controller k_position="100.0" k_velocity="40.0" soft_lower_limit="-500" soft_upper_limit="500"/>
+    <parent link="base_link_x"/>
+    <child link="base_link_y"/>
+    <origin rpy="0 0 0" xyz="0 0 0"/>
+    <axis xyz="0 1 0"/>
+    <dynamics damping="100.0"/>
+    <limit effort="1000" lower="-10" stiffness="1000" upper="10" velocity="2"/>
+  </joint>
+  <joint name="omni_joint_theta" type="revolute">
+    <safety_controller k_position="100.0" k_velocity="40.0" soft_lower_limit="-500" soft_upper_limit="500"/>
+    <parent link="base_link_y"/>
+    <child link="chassis_link"/>
+    <origin rpy="0 0 0" xyz="0 0 0"/>
+    <axis xyz="0 0 1"/>
+    <dynamics damping="100.0"/>
+    <limit effort="1000" lower="-6.283184" stiffness="1000" upper="6.283184" velocity="2"/>
+  </joint>
+  <!-- ==================== -->
+  <!-- Dingo links          -->
+  <!-- ==================== -->
+  <!-- Chassis link -->
+  <link name="chassis_link">
+    <visual>
+      <origin rpy="0 0 0" xyz="0 0 0"/>
+      <geometry>
+        <mesh filename="../meshes/base/omni_chassis.dae"/>
```

Comment:
`chassis_link` uses .dae file, which is not the supported format in mujoco or yourdfpy. Should I leave it or should I just use `omni_chassis_collision.stl`?

Reply:
Please use the `stl`-files instead.